### ISSUE TITLE
$GITHUB_OUTPUT should not require double quoting

### ISSUE THREAD

**Issue by jonashaag** [rhysd/actionlint#470]
```yml
on:
  push:

jobs:
  test:
    runs-on: ubuntu-latest
    steps:
      - run: echo "x=y" >> $GITHUB_OUTPUT
```

```
shellcheck reported issue in this script: SC2086:info:1:15: Double quote to prevent globbing and word splitting [shellcheck]
```

It should be safe to assume that `$GITHUB_OUTPUT` will never contain any characters that are relevant to globbing and splitting so IMO it should be fine to not quote it.
